source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -441,6 +443,50 @@ Example:
 
   <queue_size>16384</queue_size>
 
+.. _reference_agents_disconnection_time:
+
+agents_disconnection_time
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.1.0
+
+This sets the time since the last keepalive of an agent to consider it as disconnected.
+
++-------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| **Default value**       | 20s                                                                                                                               |
++-------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| **Allowed values**      | A positive number that should end with a character indicating a time unit, such as: s (seconds), m (minutes), h (hours), d (days) |
++-------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+
+Example:
+
+.. code-block:: xml
+
+  <agents_disconnection_time>1m</agents_disconnection_time>
+
+.. _reference_agents_disconnection_alert_time:
+
+agents_disconnection_alert_time
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.1.0
+
+This sets the time since an agent is considered disconnected to trigger an alert.
+Like this time is added to the :ref:`agents_disconnection_time<reference_agents_disconnection_time>` parameter, the minimun time frame to produce an alert is 2m and 20s.
+Also consider that the daemon :doc:`monitord<../../reference/daemons/ossec-monitord>` checks the status of the agents evey two minutes, so even when the neccesary time to trigeer the alert has passed, it's neccesary to wait this scan to begin.
```

Comment:
For now, remove this clarification until we discuss how to handle this scenario.

Reply:
Removed!